libs/sr/ui/src/LightCone/LightConeEditor/LightConeEditor.tsx

```diff
@@ -124,7 +123,7 @@ export function LightConeEditor({
   }, [lightConeIdToEdit])
 
   return (
-    <Container>
+    <Suspense fallback={false}>
```

Comment:
Probably a good idea to put something in here, since using `fallback=false` would cause layout shift as element is loaded. I am fine leaving this here for now since we are likely to put this element in a modal later.